source/usage-openstack.rst

```diff
@@ -100,13 +81,13 @@ You can refresh the access token and obtain an OpenStack token in a single
 :download:`../get-token.py` script expecting your credentials to be available in
 the environment:
 
-* CHECKIN_CLIENT_ID: Your Check-in client id (get it from https://aai.egi.eu/fedcloud)
-* CHECKIN_CLIENT_SECRET: Your Check-in client secret (get it from https://aai.egi.eu/fedcloud)
-* CHECKIN_REFRESH_TOKEN: Your Check-in refresh token (get it from https://aai.egi.eu/fedcloud)
-* OS_AUTH_URL: Keystone URL (depends on the provider, you can get it in https://goc.egi.eu)
-* OS_PROJECT_ID: OpenStack project to use (See script above for obtaining it)
+* ``CHECKIN_CLIENT_ID``: Your Check-in client id (get it from https://aai.egi.eu/fedcloud)
```

Comment:
Here also using proper links may be better.

Reply:
Also fixed